docs/docs/60-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -302,11 +302,12 @@ as native processes.
 
     If you have a paid ngrok account that allows you to use a custom domain name
     for your tunnels, you can specify that domain name using the
-    `KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME` environment variable before running
+    `KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME` and (to properly set the protocol scheme) `KARGO_EXTERNAL_WEBHOOKS_SERVER_TLS_TERMINATED_UPSTREAM`  environment variables before running
     the make target:
```

Comment:
We try to observe a soft limit of 80 chars and hard(er) limit of 120... easier to do code/doc reviews on a mobile device.

(If you're using vscode, I find `"editor.rulers": [80, 120]` to be a helpful setting.)

```suggestion
    `KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME` and (to properly set the protocol
    scheme) `KARGO_EXTERNAL_WEBHOOKS_SERVER_TLS_TERMINATED_UPSTREAM` environment
    variables before running the make target:
```